.github/workflows/app-build-verify.yml

```diff
@@ -111,11 +108,14 @@ jobs:
         path: ${{ steps.config.outputs.briefcase-data-dir }}
 
     - name: Set Up Python
-      # Linux System apps requires python is System Python to run the app
-      if: ${{ !startsWith(inputs.runner-os, 'ubuntu') || !startsWith(inputs.python-version, steps.config.outputs.system-python-version) }}
+      # On Linux, accept "system" as a proxy for "don't install Python"; this will
+      # fall back to the system-provided Python package. Providing "system" as a value
+      # on macOS or Windows will cause an error.
+      if: ${{ !startsWith(inputs.runner-os, 'ubuntu') || inputs.python-version != 'system' }}
       uses: actions/setup-python@v5.2.0
       with:
-        python-version: ${{ inputs.python-version }}
+        python-version: ${{ inputs.python-version || '3.x' }}
```

Comment:
By way of explanation: It appears "with: ''" is passed through as a literal empty string, rather than falling back to a default, so we need to provide the default as a fallback here, as well (same for the create-verify analog).